content/en/docs/reference/command-line-tools-reference/feature-gates/watch-list.md

```diff
@@ -9,5 +9,8 @@ stages:
   - stage: alpha 
     defaultValue: false
     fromVersion: "1.27"
+  - stage: beta
```

Comment:
`toVersion` should be added to the previous stage.

```suggestion
    toVersion: "1.31"
  - stage: beta
```

Reply:
fixed, thanks.